appengine/go11x/pubsub/authenicated_push/README.md

```diff
@@ -0,0 +1,94 @@
+#  Pub/Sub sample for Google App Engine Standard Environment
```

Comment:
We've been moving away from in-depth READMEs in favor of having a single version of the docs on cloud.google.com. We've had issues keeping READMEs up to date. Is this the canonical version or is there another page we can point to?

Reply:
I am not sure... I had copied the structure of the READMEs for python/node, and adapted for Go. On the page this sample is used their are no directions about deploying the app or running locally. I guess I am not sure how to proceed with this. I can remove it if that is the right thing to do, but I feel like there is some valuable information here. Thoughts?

Reply:
I believe most users are viewing this from the docs on cloud.google.com so having more information here is probably bad. @codyoss, is the page you're referring to [this link](https://cloud.google.com/pubsub/docs/push#authentication_and_authorization_by_the_push_endpoint)?

Reply:
Yes it is.